### ISSUE TITLE
License requirements for landing on a planet

### ISSUE THREAD

**Issue by esksp** [endless-sky/endless-sky#10594]
### Problem Description

It would be useful to add a license requirement for landing on a planet. Think of it as a visa...
For instance, paradise planets should have expensive visas (you won't let any scum land on this exclusive and utterly expensive place, would you). Some others are "members only".

In my case, I have an organization which would only let you land on their station if you are accepted by them, no matter how much brown-nosing you might do. In other terms, you could be the single most popular person for them, as long as you aren't on the guest list, you stay out, period.

### Related Issue Links

N/A

### Desired Solution

See above

### Alternative Approaches

N/A

**Comment by TomGoodIdea** [comment]
would be possible with #9599

**Comment by esksp** [comment]
Indeed, this is a subset of #9599, assuming the access conditions set allows for licenses.

**Comment by TomGoodIdea** [comment]
> Indeed, this is a subset of #9599, assuming the access conditions set allows for licenses.

licenses are just simple conditions in save files, so yeah, it would be totally possible
